typings/index.d.ts

```diff
@@ -1305,9 +1282,9 @@ declare module 'klasa' {
 		public value: any;
 		public is: string;
 
-		private parent: ?Type;
-		private childKeys: Map;
-		private childValues: Map;
+		private parent: Type | null;
```

Comment:
the ? makes the type nullable, although it was in the wrong place, it should be `private parent?: Type;`

Reply:
Ah okay. I would have thought a `?` on the property name would mean the property itself might not exist.